cartographer/mapping/internal/optimization/optimization_problem_2d.cc

```diff
@@ -175,7 +175,9 @@ OptimizationProblem2D::~OptimizationProblem2D() {}
 
 void OptimizationProblem2D::AddImuData(const int trajectory_id,
                                        const sensor::ImuData& imu_data) {
-  imu_data_.Append(trajectory_id, imu_data);
+  // TODO(MichaelGrupp): IMU data is currently not used in 2D optimization
```

Comment:
A `TODO` should be actionable. How about just leaving a comment that this satisfies the interface but we do not want to keep IMU data?

We should then also remove `imu_data_` if possible.